gui/src/components/CheckboxButton.svelte

```diff
@@ -0,0 +1,81 @@
+<script lang="ts">
+  export let active: boolean = false;
+  export let tooltip: string = '';
+
+  import Show from './mono/check.svelte';
```

Comment:
nit: imports should go at the top of the script block. I'm actually surprised this compiles at all, since this means that the import side-effects are potentially not statically analyzable.

Reply:
Thanks for catching this!